egs/multi_en/s5/local/ami_split_dev.final

```diff
@@ -0,0 +1,18 @@
+ES2011a
+ES2011b
+ES2011c
```

Comment:
I believe this file ami_split_dev.final and other .final files were not checked in originally, but automatically generated via a script from other files.
They should never have been in the local/ directory in the first place-- it's reserved for things that are checked in.  I fixed this in the AMI s5b recipe.
You should be able to remove this file from the repo.